packages/components/src/components/a11y.css

```diff
@@ -7,6 +7,13 @@
 		 * Minimum size of interactive elements.
 		 */
 		--a11y-min-size: 44px;
+		/*
+		 * No element should be used without a background and font color whose contrast ratio has
+		 * not been checked. By initially setting the background color to white and the font color
+		 * to black, the contrast ratio is ensured and explicit adjustment is forced.
+		 */
+		background-color: white;
+		color: black;
```

Comment:
- kol-icon

Reply:
Ich glaube die Snapshot-Tests laufen durch, weil (fast?) alle Beispiele einen weißen Hintergrund haben.

Für Anwendungen mit einem anderen (z.B. hellgrauen) Hintergrund fürchte ich, dass man bei sehr vielen Komponenten die `background-color` zurück auf transparent setzen müsste, auch wenn diese ihre Farben eigentlich korrekt spezifiziert haben.

Vorschlag: Statt die background-color global auf `:host` zu setzen, könnten wir sie in den Component-Styles, dort wo es Sinn macht, individuell auf das "visuelle root element" setzen. Bei Button wäre das z.B. `kol-span-wc.button-inner`.

Zur Veranschaulichung, ein paar Beispiele bei einer App mit dunklem Hintergrund:

<img width="372" alt="image" src="https://github.com/public-ui/kolibri/assets/971072/e25e32c6-20af-475b-9563-9669038a69a3">
<img width="380" alt="image" src="https://github.com/public-ui/kolibri/assets/971072/587985e5-afc8-4480-816e-3be61b0a7a49">
<img width="406" alt="image" src="https://github.com/public-ui/kolibri/assets/971072/0d505535-afd5-4d9e-a11f-caaed4f1491f">